puppet/zulip/manifests/postgres_common.pp

```diff
@@ -17,6 +12,15 @@
                         ]
   zulip::safepackage { $postgres_packages: ensure => 'installed' }
 
+  exec {'pip2_deps':
+    # Python modules used in our monitoring/worker threads
+    command => '/usr/bin/pip2 install pytz python-dateutil'
+  }
+
+  exec {'pip3_deps':
+    command => 'python3 -m pip install pytz python-dateutil'
```

Comment:
Why are these different from each other?  Also, I'm not actually convinced we want to migrate to a virtualenv for these; if CentOS has packages for these Python modules, installing them there instead would be better.

Reply:
Actually, these are not virtualenv install, but pip install to system python library. Additionally, I checked that CentOS 7 doesn't a python36 version of tz and dateutil, so pip is the only way to get them.

Reply:
OK.